docs/Manual/Deployment/Kubernetes/DeploymentResource.md

```diff
@@ -102,7 +102,7 @@ This setting specifies the type of storage engine used for all servers
 in the cluster.
 Possible values are:
 
-- `MMFiles` To use the MMfiles storage engine.
+- `MMFiles` To use the MMFiles storage engine.
```

Comment:
I don't like this change. IMO it becomes confusing, since the possible value is case sensitive.
So even if the rest of the documentation uses `MMfiles`, I'll still prefer to keep `MMFiles` here.

Reply:
I changed it from MMfiles to MMFiles, isn't that what you prefer? In the API, the storage engine names are all lower case IIRC, but we use MMFiles and RocksDB in the docs in most places if it doesn't have to be verbatim.

Reply:
Sorry, my mistake